dll/win32/browseui/shellbrowser.cpp

```diff
@@ -2387,12 +2388,33 @@ HRESULT STDMETHODCALLTYPE CShellBrowser::QueryService(REFGUID guidService, REFII
     return E_NOINTERFACE;
 }
 
+BOOL CShellBrowser::_DoesPidlRoam(LPCITEMIDLIST pidl)
+{
+    WCHAR szPath[MAX_PATH];
+    return SHGetPathFromIDListWrapW(pidl, szPath) && PathIsUNCW(szPath);
```

Comment:
Would it make sense to create something like `_ILIsNetworkPlaces` that can be used?
We have f.e. `BOOL _ILIsBitBucket(LPCITEMIDLIST pidl)` which does exactly the same, but for the bitbucket.

That would save translating the pidl to string, and parsing that string.
@yagoulas any input on this?

Reply:
Hmm, there is no such function named `_ILIsNetworkPlaces`.
The target is PIDL, so `IL...` functions can be used.